core_v2/sources/domains.move

```diff
@@ -1173,6 +1231,21 @@ module aptos_names_v2::domains {
         timestamp::now_seconds() >= expiration_time_sec
     }
 
+    /// Given signer, target address and transfer to address, return true if we should set name address and reverse lookup, false otherwise
+    public fun should_set_reverse_lookup_and_target_address(
```

Comment:
should it be target address or name address? what's the difference between setting name address and reverse lookup?

Reply:
name_address is the same thing as target_address, they are used interchangeablely in the code